src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -1417,6 +1430,18 @@ public void setPrivate(boolean value) throws IOException {
         set().private_(value);
     }
 
+    /**
+     * Sets repository template
+     *
+     * @param value
+     *            the GitHub repository used as a template
+     * @throws IOException
+     *             the io exception
+     */
+    public void setTemplateRepository(GHRepository value) throws IOException {
+        set().templateRepository(value);
+    }
+
```

Comment:
Can you actually change the template repository after it being set at creation?

Reply:
Deleted this method, As you are saying it doesn't have sense